### DISCUSSION TITLE
Ferramentas para melhorar seu CSS

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#525] [Dicas]
pessoal escrevi um artigo no Medium , de algumas ferramentas que usei na refatoração do código CSS da Printi, caso queiram ler, e /ou opinar sobre o artigo, opiniões e sugestões seram bem-vindas.

Podem ler o artigo [aqui](https://medium.com/@rubenmarcus/ferramentas-para-melhorar-o-seu-css-6afa79ef5206)

Obrigado

**Comment by heliomsolivas** [comment]
Maneiro, vou provavelmente adotar algo do seu artigo, no sublime text eu utilizo o **CSSComb** pra organizar as propriedades (no meu caso em ordem alfabética) e acho uma maravilha.

**Comment by rubenmarcus** [comment]
Legal @heliomsolivas  , o CSS Comb é mais por questão de organização certo? em questão de performance,o impacto é o mesmo se voce começar um elemento com width, ou termina-lo com width? (Estou falando bobagem ?)

div{ background:#fff; width:20px }  é o mesmo pro browser que div{width:20px; background:#fff} ??

acho que visualmente o css comb e essa organização é mais bonita,mas eu já tenho o hábito e um padrão meu mesmo, que eu coloco primeiro as propriedades espaciais, e depois as de estilo..

**Comment by kvnol** [comment]
@rubenmarcus curti o post e já comecei a utilizar o CSS Lint no Sublime, mesmo utilizando SASS em todos os projetos.
